### ISSUE TITLE
Terraform 0.13.1 prompting for variables twice

### ISSUE THREAD

**Issue by Neutrollized** [hashicorp/terraform#26027]
### Terraform Version

```
0.13.1
```

### Terraform Configuration Files

Irrelevant, however it happens with S3 backend as well as local backend (from my testing)

### Debug Output

[terraform 0.13.1](https://gist.github.com/Neutrollized/d97b1942314b7312b66725265e2cc645)
vs
[terraform 0.13.0](https://gist.github.com/Neutrollized/2cc8619ac62dbbb2957e42d82f26abbf)

### Crash Output

EDIT 2: [Crash](https://gist.github.com/Neutrollized/148feb55a15fe1297809132ca9db3e6f) only happens if I try to **ctrl+c** out of the re-prompt (also mentioned in comment below)

### Expected Behavior

Prompt only once

### Actual Behavior

Prompted for (all) variables twice.

### Steps to Reproduce

`terraform init`, `terraform plan` and `terraform apply` both would prompt for variables twice.

### Additional Context

It prompts for all variables twice.  So if you have 3 vars that need to be prompted for input, it will re-prompt you for all 3 in the same order.  If you have vars in `terraform.tfvars` defined, then it works as intended and there's no prompting.

EDIT: aside from prompting twice, everything builds (and destroys) as intended.

### References

Not a GitHub reference, but does Reddit count? :D https://www.reddit.com/r/Terraform/comments/ihs6hx/terraform_asking_for_input_variable_values_twice/

**Comment by mildwonkey** [comment]
thanks for reporting this @Neutrollized !

**Comment by Neutrollized** [comment]
Holy crap.  I thought your name looked familiar; I saw you give the talk on journey to Terraform 1.0 during HashiConf EU!  (Beautiful cat btw)

**Comment by Neutrollized** [comment]
Just thought I'd do some more testing on my own and here's some additional info that might be useful to you:

1.  my original terraform files included a backend.tf which used s3 as the TF state backend, but I get the same double prompt bug even if I were to use local state
2.  if I **ctrl+c** out of a `terraform plan` (as an example) on the first prompt, I get the expected error of:
```
var.vpc_id
  Enter a value: ^C
Interrupt received.
Please wait for Terraform to exit or data loss may occur.
Gracefully shutting down...

Error: No value for required variable

  on variables.tf line 4:
   4: variable "vpc_id" {}

The root module input variable "vpc_id" is not set, and has no default value.
Use a -var or -var-file command line argument to provide a value for this
variable
```
However...If I were to **ctrl-c** on the re-prompt, I get a Terraform crash instead:
```
var.vpc_id
  Enter a value: vpc-yyyyyyyyyyyyyyyyyy

var.vpc_id
  Enter a value: ^C
Interrupt received.
Please wait for Terraform to exit or data loss may occur.
Gracefully shutting down...
panic: runtime error: invalid memory address or nil pointer dereference
...
SECURITY WARNING: the "crash.log" file that was created may contain
sensitive information that must be redacted before it is safe to share
on the issue tracker.

[1]: https://github.com/hashicorp/terraform/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
Details from crash log can be found [here](https://gist.github.com/Neutrollized/148feb55a15fe1297809132ca9db3e6f)

Hope this helps!

**Comment by alisdair** [comment]
I believe this issue was fixed in #26063, but GitHub didn't auto-close it. This is part of today's 0.13.2 release.
